### ISSUE TITLE
Doesn't support text decoration.

### ISSUE THREAD

**Issue by sushmitabajgain** [puzzle/prawn-markup#31]
When I try to add links and underline or color text and  it doesn't support any.

**Comment by codez** [comment]
Links and underline should work, color just got added in #29. Could you please give an example of the markup that does not work?

**Comment by allthesignals** [comment]
I ran a little test to check and this seems to work okay:
```html
hello <a href="https://google.com"><color rgb="ff0000">world</color></a>
```

[test.pdf](https://github.com/puzzle/prawn-markup/files/8227822/test.pdf)

**Comment by allthesignals** [comment]
> Links and underline should work, color just got added in #29. Could you please give an example of the markup that does not work?



> Links and underline should work, color just got added in #29. Could you please give an example of the markup that does not work?

Might need to make a new release?

**Comment by allthesignals** [comment]
> Links and underline should work, color just got added in #29. Could you please give an example of the markup that does not work?



> Links and underline should work, color just got added in #29. Could you please give an example of the markup that does not work?

Might need to make a new release?

**Comment by codez** [comment]
Just released 0.3.5
